Silent rm error for sccache log file

zhouzhuojie opened the pull request:
Sample reporting from dr.ci

![image](https://user-images.githubusercontent.com/658840/131724645-75afa04f-7554-4674-8e7c-cf139c84d994.png)


The `rm` command is not actually running into problems, just need to silent the console output.

malfet commented on .jenkins/pytorch/common.sh:
```diff
@@ -74,7 +74,7 @@ if [[ "$BUILD_ENVIRONMENT" != *win-* ]]; then
     # Save sccache logs to file
     sccache --stop-server > /dev/null  2>&1 || true
-    rm ~/sccache_error.log || true
+    rm -f ~/sccache_error.log
```
Please keep `|| true` in place, as we do not want to error out if log can not be deleted
```suggestion
    rm -f ~/sccache_error.log || true
```

zhouzhuojie committed:
Silent rm error for sccache log file (#64388)

Summary:
Sample reporting from dr.ci

![image](https://user-images.githubusercontent.com/658840/131724645-75afa04f-7554-4674-8e7c-cf139c84d994.png)

The `rm` command is not actually running into problems, just need to silent the console output.

Pull Request resolved: https://github.com/pytorch/pytorch/pull/64388

Reviewed By: walterddr, malfet, seemethere

Differential Revision: D30704439

Pulled By: zhouzhuojie

fbshipit-source-id: 7a007fe858a2208ee209e6d2f13fc2b33f9c0012
